charts/gsp-cluster/templates/02-gsp-system/service-operator/service.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/port: "8443"
+    prometheus.io/scheme: https
+    prometheus.io/scrape: "true"
```

Comment:
see above - if the service operator isn't under istio management, i don't think this will do anything

Reply:
_wontfix_